ee/codegen/src/__test__/project.test.ts

```diff
@@ -177,6 +177,95 @@ describe("WorkflowProjectGenerator", () => {
       }
     );
   });
+  describe.skip("generateCodeNodeDirectoryOnly", () => {
+    const excludeFilesAtPaths: RegExp[] = [/\.pyc$/];
+    const ignoreContentsOfFilesAtPaths: RegExp[] = [];
+
+    it.each(
+      getFixturesForProjectTest({
+        includeFixtures: [""],
+        fixtureMocks: {},
+      })
+    )(
+      "should correctly generate code without display directory for fixture $fixtureName",
```

Comment:
I'm having a hard time grokking the goal here:
- doesn't `includeFixtures` mean that none are included?
- What's the codegen bug we are hoping to catch here? I thought it was a serialization bug